src/RingGrids/grids/octahedral_gaussian.jl

```diff
@@ -0,0 +1,118 @@
+"""
+    G = OctahedralGaussianGrid{T}
+
+An Octahedral Gaussian grid that uses `nlat` Gaussian latitudes, but a decreasing number of longitude
+points per latitude ring towards the poles. Starting with 20 equi-spaced longitude points (starting at 0˚E)
+on the rings around the poles, each latitude ring towards the equator has consecuitively 4 more points,
+one for each face of the octahedron. E.g. 20, 24, 28, 32, ...nlon-4, nlon, nlon, nlon-4, ..., 32, 28, 24, 20.
+The maximum number of longitue points is `nlon`. The values of all grid points are stored in a vector
+field `v` that unravels the data 0 to 360˚, then ring by ring, which are sorted north to south."""
+struct OctahedralGaussianGrid{T} <: AbstractOctahedralGrid{T}
+    data::Vector{T}    # data vector, ring by ring, north to south
+    nlat_half::Int  # number of latitudes on one hemisphere
+
```

Comment:
This doesn't have the `rings` object yet, or is it not needed because the indices are easier to compute for the Octahedral grids?

Reply:
Now it does ;) hadn't tackled that yet when you reviewed